stable/minio/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Minio is a high performance distributed object storage server, designed for large-scale private cloud infrastructure.
 name: minio
-version: 1.6.5
+version: 1.6.6
```

Comment:
new features 1.7.0? I think this one is ready to go now - great PR

Reply:
cool, thanks! PTAL